src/Providers/Maps/Google/Features/Shape.ts

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Provider.Maps.Google.Feature {
+    export class Shape
+        implements
+            OSFramework.Maps.Feature.IShape,
+            OSFramework.Maps.Interface.IBuilder,
```

Comment:
I think we don't need to implement this!

Reply:
Because of the FeatureBuilder, we need this. They need to create the mecanism to expose the feature for both maps.

Reply:
Ok, lets maintain the build and delete the disposable. You can look at offset feature. it has the build smaller using an eslint comment.